shared/constants/chat2/meta.js

```diff
@@ -135,22 +135,34 @@ export const updateMeta = (
     ? old.resetParticipants
     : meta.resetParticipants
 
-  return meta.withMutations(m => {
+  let base = meta
```

Comment:
I'm a little iffy on this part. A newer untrusted meta was wiping out data in an existing trusted meta. This switches the merge direction in that case if the inbox versions are the same.